pos_receipt_hide_user/static/src/xml/hide_user_button.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+  <t t-name="pos_receipt_hide_user.hide_user" owl="1">
+    <div class="button hide_user" t-on-click="hideUser">
```

Comment:
contrarily to `pos_receipt_hide_price`, in that case we will not add a button: you can apply immediately the configured option